js/dice-overlay.js

```
@@ -74,58 +74,133 @@ let bonusMsg = null;
+          hunger: parseInt(element.querySelector("#hungerInput").value) || 0,
+          rouse: parseInt(element.querySelector("#rouseInput").value) || 0,
+          remorse: parseInt(element.querySelector("#remorseInput").value) || 0,
+          frenzy: parseInt(element.querySelector("#frenzyInput").value) || 0,
+          difficulty: parseInt(element.querySelector("#difficultyInput").value) || 1,
+        };
+
+        // ----------------------------------------------------
+        //  Blood Surge – adds dice & a mandatory Rouse check
+        // ----------------------------------------------------
+        const bloodSurgeToggle = element.querySelector("#bloodSurgeToggle");
+        const bloodSurgeOn = bloodSurgeToggle && bloodSurgeToggle.checked;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)